config/example.yaml

```diff
@@ -13,6 +13,7 @@ Google:  # only relevant when using the oauth authbackend
 GitHub:  # only relevant when using the oauth authbackend
   ClientID: replace me
   ClientSecret: replace me
+GitHubEndpointURL: # (OPTIONA) URL for custom GitHub endpoint (github enterprise); e.g. 'https://github.mydomain.com'
```

Comment:
Rather than defining a new top-level configuration option, this should be an `EndpointURL` option inside the `GitHub` section. See below for more.

Also, nit: missing an `L` at the end of `OPTIONAL`

Reply:
done!